src/app/components/results-menu/results-menu.component.html

```diff
@@ -6,18 +6,18 @@
     <div
       id="result-div"
       class="w100"
-      [resizeEdges]="{ top: !(imageViewerOpen$ | async) }"
+
       [ngClass]="{'mobile-size': breakpoint === breakpoints.MOBILE}"
       [style.height.px]="menuHeightPx"
       mwlResizable
       [validateResize]="validate"
       [enableGhostResize]="true"
-      [resizeCursorPrecision]="20"
```

Comment:
This gets rid of the cursor on the edge of the results menu. Should probably either stay or move down as well